src/odemis/odemisd/main.py

```
@@ -400,6 +400,9 @@ def _terminate_independent(self, comp):
             children = comp.children.value
             for child in children:
                 self._terminate_component(child)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)